certified-connectors/Leadinfo/apiDefinition.swagger.json

```diff
@@ -0,0 +1,293 @@
+              "description": "page_url_first_visit"
+            },
+            "branch_code": {
+              "type": "string",
+              "description": "branch_code"
+            },
+            "branch_code_international": {
+              "type": "string",
+              "description": "branch_code_international"
+            }
+          }
+        },
+        "description": "webhook response"
+      },
+      "put": {
+        "summary": "Triggers when a company is send",
+        "description": "Triggers when a company is send",
+        "responses": {
+          "201": {
+            "description": "success"
+          }
+        },
+        "operationId": "Webhook-company-trigger",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                },
+                "integration": {
+                  "type": "string",
+                  "description": "integration",
+                  "title": "",
+                  "x-ms-visibility": "internal",
+                  "default": "microsoftpowerautomate"
+                },
+                "type": {
+                  "type": "string",
+                  "description": "Flow type",
+                  "title": "Type",
+                  "default": "share",
+                  "enum": [
+                    "share",
+                    "trigger"
+                  ]
+                }
+              },
+              "required": [
+                "url",
+                "integration",
+                "type"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/v1/webhook/{id}": {
+      "delete": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "x-ms-visibility": "internal",
+        "summary": "Delete webhook",
+        "operationId": "Delete-webhook",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2-auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://portal.leadinfo.com/oauth2",
+      "tokenUrl": "https://api.leadinfo.com/oauth2/token",
+      "scopes": {
+        "webhook": "webhook"
+      }
+    }
+  },
+  "security": [
+    {
+      "oauth2-auth": [
+        "webhook"
+      ]
+    }
+  ],
+  "tags": []
```

Comment:
@GerbenZ01 please add the "x-ms-connector-metadata" property to the swagger file. Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json#:~:text=%22x%2Dms%2Dconnector,%5D).